BaseStation/MainWindow/UserInterface.csproj

```diff
@@ -5,7 +5,7 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{722B3498-9988-4667-B02C-C9D15D288657}</ProjectGuid>
-    <OutputType>WinExe</OutputType>
+    <OutputType>Library</OutputType>
```

Comment:
The user interface is now started from the HuskyRobotics.BaseStation.StartBaseStation class and no longer needs to have an entry point.